node/src/components/linear_chain_sync.rs

```diff
@@ -622,3 +708,45 @@ where
             move || Event::GetBlockHeightResult(block_height, BlockByHeightResult::Absent(cloned)),
         )
 }
+
+/// Returns key in the Global State under which the LinearChainSync's state is stored.
+fn state_key(chainspec: &Chainspec) -> Vec<u8> {
+    chainspec.network_config.name.clone().into()
+}
+
+/// Deserialized vector of bytes into `LinearChainSync::State`.
+/// Panics on deserialization errors.
+fn deserialize_state(serialized_state: &[u8]) -> Option<State> {
+    bincode::deserialize(&serialized_state).unwrap_or_else(|error| {
+        panic!(
+            "could not deserialize state from storage type name err {:?}",
```

Comment:
```suggestion
            "could not deserialize state from storage type name, error: {:?}",
```
Or just:
```suggestion
            "could not deserialize state from storage type name: {:?}",
```

Reply:
Done in [`6d6256a` (#904)](https://github.com/CasperLabs/casper-node/pull/904/commits/6d6256afbfb52e3c3bcc4ea817550a3ce25cea36).